### PR TITLE
Added solution for diffusion reaction equation

### PR DESCRIPTION
**Pull request by bfan05**

### CODE REVIEW

**Comment by lululxvi** [comment]
In the first line, add the supported backend.

File: examples/diffusion_reaction.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+data = dde.data.TimePDE(geomtime, pde, [], num_domain=320, solution=func, num_test=80000)
+ 
+layer_size = [2] + [30] * 6 + [1]
+activation = "tanh"
+initializer = "Glorot uniform"
+net = dde.maps.FNN(layer_size, activation, initializer)
+ 
+net.apply_output_transform(
+    # Backend tensorflow.compact.v1 or tensorflow
+    lambda x, y: x[:, 1:2] * (np.pi ** 2 - x[:, 0:1] ** 2) * y
```
Comment (lululxvi) [inline examples/diffusion_reaction.py:60]: Write a function. Do not use lambda here

**Comment by lululxvi** [comment]
What is the accuracy you obtained?

**Comment by bfan05** [comment]
L2 Relative Error was 1.62e-03

**Review by bfan05 — COMMENTED** [review]
Meant to write edited diffusion_reaction.py

File: examples/diffusion_reaction.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+import deepxde as dde
+import numpy as np
+ 
+# Backend tensorflow.compact.v1 or tensorflow
+from deepxde.backend import tf
+
+# Backend pytorch
+# import torch
+ 
+def pde(x, y):
```
Comment (lululxvi) [inline examples/diffusion_reaction.py:11]: Add comment: # Backend pytorch

**Comment by bfan05** [comment]
Do you mean # Backend tensorflow.compact.v1 or tensorflow? Because # Backend pytorch is already there, or should I add it again in front of those codes?

**Comment by lululxvi** [comment]
Yes

File: docs/demos/forward.rst [hunk 3]
```diff
@@ -41,6 +41,7 @@ Time-dependent PDEs
    burgers.rar
 
 - `Heat equation <https://github.com/lululxvi/deepxde/blob/master/examples/heat_conduction_1d_uniform_bar.ipynb>`_
+- `Diffusion-reaction equation <https://github.com/bfan05/deepxde/blob/diffusion_reaction_add_code/examples/diffusion_reaction.py>`_
```
Comment (lululxvi) [inline docs/demos/forward.rst:44]: Not https://github.com/bfan05 --- lululxvi
